### PR TITLE
Concept for prefix system

### PR DESCRIPTION
**Pull request by Minenash**
A proposal for what the prefix system could be

### CODE REVIEW

File: src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java [hunk 1]
```diff
@@ -24,4 +26,15 @@ public sealed interface ResultEntry permits InvokeResultEntry, SetSearchTextEntr
      * @return the text of this entry
      */
     Text text();
+
+    /**
+     * @return the prefix used to identify the extension in the search result.
+     * @apiNote By default, it'll use the translation if it exists, if it doesn't, it'll use the extension's name
+     * */
+    default Text prefix() {
+        String key = Util.createTranslationKey("limelightExtension", extension().id()) + ".prefix";
```
Comment (BasiqueEvangelist) [inline src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java]: tbh it feels like there should be a common method for getting the `limelightExtension.namespace.id` part

**Review by BasiqueEvangelist — APPROVED** [review]

File: src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java (hunk 1 above)
Comment (Minenash) [reply src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java]: like a "LimelightExtention.translationKey()"?

Comment (BasiqueEvangelist) [reply src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java]: `baseTranslationKey` probably, but yeah

Comment (Minenash) [reply src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java]: Sorry for the delay, github didn't update the page with your reply

**Pull request merged**